test/infrastructure/docker/docker/machine.go

```diff
@@ -325,9 +328,21 @@ func (m *Machine) ExecBootstrap(ctx context.Context, data string) error {
 		return errors.Wrap(err, "failed to decode machine's bootstrap data")
 	}
 
-	commands, err := cloudinit.Commands(cloudConfig)
+	var commander provisioning.Commander
+
+	switch format {
+	// If format is empty, assume the format is cloud-init.
+	case "", bootstrapv1.CloudConfig:
```

Comment:
```suggestion
	var commander provisioning.Commander
	switch format {
	case bootstrapv1.CloudConfig:
```
I think it's fine to not handle the empty string as we already default in getBootstrapData